angular/.angular-cli.json

```diff
@@ -34,7 +34,8 @@
         "prod": "environments/environment.prod.ts",
         "prodcompose": "environments/environment.prodcompose.ts",
         "node": "environments/environment.node.ts",
-        "net": "environments/environment.net.ts"
+        "net": "environments/environment.net.ts",
+        "openshift": "environments/environment.openshift.ts"
```

Comment:
why arent we using environment.prod.ts and instead making another new file? this configuration should be valid not only for docker but also for other docker deployments